src/components/Select/Select.jsx

```diff
@@ -1,18 +1,37 @@
 import React from "react";
 import ReactSelect, { components } from "react-select";
+import AsyncSelect from "react-select/async";
+import CreatableSelect from "react-select/creatable";
 import PropTypes from "prop-types";
-import { classNames } from "../../utils";
-import styles from "./styles.css";
 
 const DropdownIndicator = (props) => {
   return (
     <components.DropdownIndicator {...props}>
       <div
-        className={classNames(
-          styles.arrowAnchor,
-          props.selectProps.menuIsOpen && styles.arrowAnchorOpen
-        )}
-      />
+        style={{
+          position: "relative",
+          height: "100%",
+          width: "2rem",
+        }}>
+        <div
+          style={{
+            content: "",
+            width: "0",
+            height: "0",
+            border: "0.5rem solid transparent",
+            borderColor:
+              "var(--select-anchor-color) transparent transparent transparent",
+            position: "absolute",
+            top: "-0.3rem",
+            right: "0.3rem",
+            ...(props.selectProps.menuIsOpen && {
+              top: "-0.9rem",
+              borderColor:
+                "transparent transparent var(--color-primary) transparent",
```

Comment:
avoid using inline styling, use conditioned classes

Reply:
@amass01, thanks for your review. I'd prefer the conditioned classes too, but with them, the arrow disappears in decrediton. The divs didn't receive the class definitions. I tried to figure out why, but only this inline styling works. Now, I revisited it without result. Do you have any idea how to solve this?

Reply:
hrmm i'm not sure, I guess it related `react-select`, as the divs are wrapped with 
`components.DropdownIndicator`. I would just leave a comment about the limitation 
we can't find a  solution or a workaround. 

I see in the documentation that there is another `classNamePrefix` that might work. iirc we already
use that one.
See: https://react-select.com/styles#using-classnames